README.md

```diff
@@ -1,3 +1,10 @@
 # sps7squared
 SPS Winter 2021 Group project repository.
 
+## Contributors ✨
+
+<table>
+  <tr>
+    <td align="center"><a href="https://github.com/SYM1000"><img src="https://avatars.githubusercontent.com/u/20364366?v=4" width="100px;" alt=""/><br /><sub><b>Santiago Yeomans</b></sub></a><br /></td>
+    <td align="center"><a href="https://github.com/sammurraytuesta"><img src="https://avatars.githubusercontent.com/u/60449164?v=4" width="100px;" alt=""/><br /><sub><b>Samantha Murray</b></sub></a><br /></td>
+    <td align="center"><a href="https://github.com/rjkroege"><img src="https://avatars.githubusercontent.com/u/749383?v=4" width="100px;" alt=""/><br /><sub><b>Robert Kroeger</b></sub></a><br /></td>
```

Comment:
Maybe put Robert at the top and give him the title "Project Advisor." I'll add more to the documentation later!

Reply:
Github markup [supports tables](https://docs.github.com/en/github/writing-on-github/organizing-information-with-tables). Please use that instead.